python/lsst/ts/tunablelaser/mock_server.py

```diff
@@ -41,7 +74,7 @@ class MockServer(tcpip.OneClientServer):
         The port that the server will start on.
     """
 
-    def __init__(self, port=0) -> None:
+    def __init__(self, port=9000) -> None:
```

Comment:
Please do not hardcode the values for port, host, encoding and terminator but take them from configuration.